internal/metrics/metrics.go

```diff
@@ -32,3 +32,19 @@ type Metrics interface {
 	Latency(ctx context.Context, query Query, offset time.Duration, alignReduceType AlignReduce) (float64, error)
 	ErrorRate(ctx context.Context, query Query, offset time.Duration) (float64, error)
 }
+
+// PercentileToAlignReduce takes a percentile value maps it to a AlignReduce
+// value.
+//
+// TODO: once we start supporting any percentile value, this should not be
```

Comment:
why are we not able to support floats right away?

Reply:
Since we decided to use the REST API instead of MQL first, we don't need (and can't) support floats. The REST API, only allowed specific percentiles (99, 95, 50)

Reply:
makes sense. let's have validation that if values like 97 is specified it should return error instead of defaulting to some unwanted value.

Reply:
Done. I'm using `fmt.Errorf` since `pkg/errors` doesn't have a constructor with formatting.

Reply:
don't use fmt.Errorf. Try errors.Errorf.

Reply:
Will do so. That actually makes more sense. It's interesting that go-lint recommended `fmt.Errorf`